modules/backlogs/spec/contracts/work_packages/base_contract_spec.rb

```diff
@@ -289,15 +220,15 @@
           work_package.version = nil
         end
 
-        it_behaves_like 'is valid'
+        include_examples 'is valid'
       end
 
       describe 'WITH a version' do
         before do
           work_package.version = version1
         end
 
-        it_behaves_like 'is valid'
+        include_examples 'is valid'
```

Comment:
👍 The output reads better this way

Reply:
indeed, but I did not touch this part. I'll probably change it in a future commit.